sys-power/nvram-wakeup/nvram-wakeup-1.1-r1.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic
+
+MY_P=${P/e}
+DESCRIPTION="read and write the WakeUp time in the BIOS"
+HOMEPAGE="https://sourceforge.net/projects/nvram-wakeup"
+SRC_URI="mirror://sourceforge/nvram-wakeup/${MY_P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+S=${WORKDIR}/${MY_P}
+
+src_prepare() {
+	default
+	# Need to be careful with CFLAGS since this could eat your bios
```

Comment:
@gentoo/qa does this really make sense?

Reply:
`CFLAGS` were discussed in [bug 47162](https://bugs.gentoo.org/47162#c16) comment 16 and following.